build.sbt

```diff
@@ -132,36 +143,46 @@ lazy val cssparse = crossProject
 lazy val cssparseJVM = cssparse.jvm
 lazy val cssparseJS = cssparse.js
 
-lazy val byteparse = crossProject.dependsOn(fastparse).settings(
-   name := "byteparse"
- ).settings(shared:_*)
+lazy val byteparse = crossProject
+  .dependsOn(fastparse)
+  .settings(shared:_*)
+  .settings(
+    name := "byteparse"
+  )
 
 lazy val byteparseJVM = byteparse.jvm
 lazy val byteparseJS = byteparse.js
 
-lazy val perftests = crossProject.settings(
-   name := "perfomance-tests"
- ).settings(shared:_*).dependsOn(fastparse, pythonparse, scalaparse, cssparse, byteparse)
+lazy val perftests = crossProject
+  .dependsOn(fastparse, pythonparse, scalaparse, cssparse, byteparse)
+  .settings(shared:_*)
+  .settings(
+    name := "perfomance-tests",
+    parallelExecution := false
```

Comment:
This bit should let us run the perftests all in the same process without worrying about parallelism messing with our resutls